### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by elizoller**
**GitHub Issue**: (https://github.com/Islandora/documentation/issues/1798)

# What does this Pull Request do?
Converts to github actions
updates the badges
removes travis

# What's new?
* new CI runner

# How should this be tested?
* make sure the tests pass (i don't think they will until this PR is merged - but you can see them running over on the branch itself)
passed here: 
https://github.com/asulibraries/Crayfish/runs/2462160374?check_suite_focus=true#step:9:42
and codecov should be running as well (proof here https://codecov.io/gh/asulibraries/Crayfish/commit/67a862c0d4e086d22f2496e06e0e82e52728b6b7/)

# Interested parties
@Islandora/8-x-committers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # ![Crayfish](https://cloud.githubusercontent.com/assets/2371345/15409657/2dfb463a-1dec-11e6-9089-06df94ef3f37.png) Crayfish
 
 [![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%207.3-8892BF.svg?style=flat-square)](https://php.net/)
-[![Build Status](https://travis-ci.com/Islandora/Crayfish.svg?branch=main)](https://travis-ci.com/Islandora/Crayfish)
+[![Build Status](https://github.com/islandora/crayfish/actions/workflows/build-dev.yml/badge.svg)](https://github.com/Islandora/Crayfish/actions)
```
Comment (whikloj) [inline README.md:4]: You are not the first to do this, but I ask this question everytime. Does our `main` branch have a purpose or has the `dev` branch now just become **main** and the actual `main` branch could be turfed?

Comment (elizoller) [reply README.md:4]: Dev is the default branch on GitHub so that’s why I did that but I am probably wrong. @dannylamb ?

Comment (whikloj) [reply README.md:4]: I don't think your wrong, but if we tag a commit in `dev` as a release then I am just wondering if we even need `main` anymore? I'll merge this and we could consider removing that branch entirely. No one is working against it.

**Review by whikloj — APPROVED** [review]
👍

**Comment by whikloj** [comment]
Rather I won't merge it, but let @dannylamb know that he can once he updates the "required status" here too.

**Pull request merged**